plugins/game/lib/api/filters/KalturaUserScorePropertiesFilter.php

```diff
@@ -325,4 +317,42 @@ public function updateUserScore($score)
 		
 		return $response;
 	}
+	
+	/**
+	 * @param $filter
+	 * @return KalturaUserScorePropertiesResponse
+	 * @throws KalturaAPIException
+	 */
+	public function deleteUserScore($filter)
+	{
+		$redisWrapper = GamePlugin::initGameServicesRedisInstance();
+		if (!$redisWrapper)
+		{
+			throw new KalturaAPIException(KalturaErrors::FAILED_INIT_REDIS_INSTANCE);
+		}
+		
+		$redisKey = $this->prepareGameObjectKey();
+		
+		if (!$this->userIdEqual)
+		{
+			throw new KalturaAPIException(KalturaErrors::USER_ID_EQUAL_REQUIRED);
+		}
+		
+		
+		$this->placesAboveUser = 0;
+		$this->placesBelowUser = 0;
+		
+		$pager = new KalturaFilterPager();
+		$result = $this->getListBySpecificUserId($redisWrapper, $pager, $redisKey);
```

Comment:
instead of returning the old result you can return 1/0 in case the zrem succeeded or failed